autopvs1/v0.2.0a/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM ubuntu:20.04
+LABEL maintainer="Miguel Browmn <brownm28@chop.edu>"
+LABEL description="Docker file with modified version of autopvs1"
+
+ENV REPO_RELEASE=v0.1.0
+
+RUN apt update -y && apt install -y wget python3 python3-pip pigz git libbz2-dev liblzma-dev zlib1g-dev \ 
+&& git clone https://github.com/d3b-center/D3b-Pathogenicity-Preprocessing \
+&& cd D3b-Pathogenicity-Preprocessing && git checkout ${REPO_RELEASE} \ 
+&& mv autopvs1 ../autopvs1 && cd ../ && rm -rf D3b-Pathogenicity-Preprocessing \ 
```

Quick trick for something like this
```suggestion
ENV REPO_RELEASE=0.1.0

RUN apt update -y && apt install -y curl wget python3 python3-pip pigz git libbz2-dev liblzma-dev zlib1g-dev \ 
&& curl -sL https://github.com/d3b-center/D3b-Pathogenicity-Preprocessing/archive/refs/tags/v${REPO_RELEASE}.tar.gz | tar xz --strip-components=1 D3b-Pathogenicity-Preprocessing-${REPO_RELEASE}/autopvs1
```